app/domain/authentication/authn_iam/authenticator.rb

```diff
@@ -0,0 +1,84 @@
+require 'active_support'
+require 'active_support/core_ext'
+require 'net/ldap'
+require 'json'
+require 'aws-sdk-iam'
+require 'aws-sdk-core'
+
+
+module Authentication
+  module AuthnIam
+
+    class Authenticator
+      
```

Comment:
Looks like there might be an extra line here.

Reply:
Removed the extra line!